### DISCUSSION TITLE
REALEARN with Bitfocus Companion

### DISCUSSION THREAD

**Discussion by EdgaruitHilversum** [helgoboss/helgobox#1128] [Ideas]
I would love to see REALEARN become compatible with Bitfocus Companion, allowing us to use Stream Deck devices to control DAW functions even more efficiently. Integrating with Companion would let us easily configure Stream Deck buttons to trigger specific REALEARN actions, like MIDI mappings and macros. This would enhance workflows and offer users great flexibility. What do you all think? Are there others interested in this integration?

**Comment by zuppoii** [comment]
Yeah! It would be so nice to see that option implemented! Would love to hear (read) Benjamin's viewpoint on this request :-)

**Comment by EdgaruitHilversum** [comment]
Meanwhile, there is now an integration for Realearn with Stream Deck.
It works FANTASTICALLY.
https://youtu.be/VSrHG1HtAq0?si=MHKTvaw1Zt1B2rUG
